### PR TITLE
Multiple times  clicking of start button is bug.

### PR DESCRIPTION
**Pull request by monil25**
After clicking start button once it has to be disabled. Enable it again after clicking of stop button.

### CODE REVIEW

File: javascript/asynchronous/loops-and-intervals/setinterval-stopwatch.html [hunk 1]
```diff
@@ -47,11 +47,13 @@
       // When the start button is pressed, start running displayCount() once per second using displayInterval()
       startBtn.addEventListener('click', () => {
         stopWatch = setInterval(displayCount, 1000);
+        startBtn.disabled =true;
```
Comment (chrisdavidmills) [inline javascript/asynchronous/loops-and-intervals/setinterval-stopwatch.html]: put space on both sides of the equials sign, i.e.

```
startBtn.disabled = true;
```

File: javascript/asynchronous/loops-and-intervals/setinterval-stopwatch.html [hunk 2]
```diff
@@ -47,11 +47,13 @@
...
       });
 
       // When the stop button is pressed, clear the interval to stop the count.
       stopBtn.addEventListener('click', () => {
         clearInterval(stopWatch);
+        startBtn.disabled=false;
```
Comment (chrisdavidmills) [inline javascript/asynchronous/loops-and-intervals/setinterval-stopwatch.html]: put space on both sides of the equials sign, i.e.

```
startBtn.disabled = false;
```

File: javascript/asynchronous/loops-and-intervals/setinterval-stopwatch.html (hunk 1 above)
Comment (monil25) [inline javascript/asynchronous/loops-and-intervals/setinterval-stopwatch.html]: startBtn.disabled = true;

File: javascript/asynchronous/loops-and-intervals/setinterval-stopwatch.html (hunk 2 above)
Comment (monil25) [reply javascript/asynchronous/loops-and-intervals/setinterval-stopwatch.html]: startBtn.disabled = false;

**Pull request merged**